### DISCUSSION TITLE
Links from changelog are broken

### DISCUSSION THREAD

**Discussion by BackEndTea** [community/community#5750] [Mobile]
Discovered this on the Auth0 php sdk: https://github.com/auth0/auth0-PHP/releases/tag/8.0.0

The link to upgrade.md opens a browser with a localhost link, instead of opening the file in the app.

This is on an iPhone 11 IOS 14.7

**Comment by alcere** [comment]
Hi! Thanks for reporting! I've shared the issue with both platform teams (iOS & Android). They will evaluate and eventually fix the issue if possible.
